### ISSUE TITLE
how to setup velocity 

### ISSUE THREAD

**Issue by earlhsjks** [BenCodez/VotifierPlus#58]
how to forward votes to my servers and what key should i place in there? velocity VotifierPlus is done, but to forward in my servers don't

**Comment by earlhsjks** [comment]
private key or public?

**Comment by BenCodez** [comment]
Public key, if you using votingplugin then you don't need to follow the votifierplus forwarding at all

**Comment by earlhsjks** [comment]
spigot server need to have separate port from the primary in order to work? and use the vote port in forwarding configuration?

**Comment by BenCodez** [comment]
Yes, if your using votingplugin then you don't need that

**Comment by earlhsjks** [comment]
can votingplugin forward the vote to pinataparty?

**Comment by BenCodez** [comment]
Yes, you'll just have to install votifier on the backend and leave it unset (but enough that it loads)
